student-work/joe-wainer/readme.md

```diff
@@ -1 +1,91 @@
-# Hello! My name is Joe Wainer.
+# Joe Wainer
+
+>Adaptable professional with proven communication, writing, and technical skills. Aiming to leverage my abilities to successfully fill the Technical Writer role at your company. Frequently praised as proactive by my peers, I can be relied upon to help your company achieve its goals.
+
+:globe_with_meridians: Israel   
+:phone: +972 58-733-2525    
+:email: hello@joewainer.com   
+:file_folder: [GitHub](https://joewainer.github.io)   
+:computer: [Website](https://joewainer.com)  
+:briefcase: [LinkedIn](https://linkedin.com/in/joe-wainer)   
+
+## Experience
+_March 2022 - Present_  
```

Comment:
for all dashes which are not bullets
please use &ndash; instead of - 
it makes for cleaner code.